homeassistant/components/airzone_cloud/sensor.py

```diff
@@ -41,6 +40,7 @@
 AIDOO_SENSOR_TYPES: Final[tuple[SensorEntityDescription, ...]] = (
     SensorEntityDescription(
         device_class=SensorDeviceClass.TEMPERATURE,
+        has_entity_name=True,
```

Comment:
If they all now use the `has_entity_name`, it can be set as `_attr_has_entity_name = True` on the class, removing the need to set it on each entity description.